org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/semantictokens/TokenType.java

```diff
@@ -32,7 +33,17 @@
 	PROPERTY("property"),
 	VARIABLE("variable"),
 	PARAMETER("parameter"),
-	MODIFIER("modifier");
+	MODIFIER("modifier"),
+
+	// Java-specific token types. These should be prefixed as such,
+	// to avoid confusion with similarly named concepts in other langugages.
+	// For example, "module" typically has the same meaning as "namespace"
+	// in other languages (Rust, for example). But in Java, modules are a
+	// completely separate concept, packages being the "namespace" tokens.
+	MODULE("javaModule"),
+	TYPE_LITERAL_KEYWORD("javaClassLiteralKeyword"), // genericName taken from: https://docs.oracle.com/javase/specs/jls/se11/html/jls-15.html#jls-15.8.2
+	JAVADOC_TAG("javadocBlockTag"), // genericName taken from: https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html#block-tags
+	JAVADOC_NESTED_TAG("javadocInlineTag"); // genericName taken from: https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html#inline-tags
```

Comment:
Suggest to reuse "keyword" token.

Reply:
Takes away capabilities from theme authors, see my other comments.